lib/project_types/theme/themekit.rb

```diff
@@ -5,19 +5,29 @@ class Themekit
     class << self
       def create(ctx, password:, store:, name:)
         stat = ctx.system(THEMEKIT,
-                          "new",
+                          'new',
                           "--password=#{password}",
                           "--store=#{store}",
                           "--name=#{name}")
         stat.success?
       end
 
+      def deploy(ctx)
+        unless push(ctx)
+          ctx.abort(ctx.message('theme.deploy.push_fail'))
+        end
+        ctx.done(ctx.message('theme.deploy.info.pushed'))
+
+        stat = ctx.system(THEMEKIT, 'publish')
```

Comment:
We actually might want to add a confirm around this because it will change the live theme and that is scary.

Reply:
We ask for confirmation before running `Themekit.deploy`:
https://github.com/Shopify/shopify-app-cli/pull/883/files#diff-f6844d536bb91d3d46c217698c1d102fR11